Model-Modifier/src/scene/surface/Surface_Loop.cpp

```diff
@@ -91,13 +43,15 @@ Object Surface::LoOutputOBJ4(std::vector<glm::vec3> edgePoints)
     Obj.m_VertexPos = VertexPos; Obj.m_FaceIndices = FaceIndices;
     Obj.m_NumPolygons = NumberPolygons;
     Obj.TriangulateFaces();
+
     return Obj;
 }
 
+
 ////////// algorithms //////////
 
 // Loop subdivision surface algorithm
-Object Surface::Loop3()
+Object Surface::Loop()
```

Comment:
note the constants `0.625, 0.75` from the original Loop algorithm are preserved, there was an idea to change these for `n`-gon faces but ultimately settled on keeping these